throughput/src/bin/zn_sub_thr_stream.rs

```diff
@@ -83,26 +92,23 @@ async fn main() {
                 let interval = 1_000_000.0 / elapsed;
                 println!(
                     "zenoh-net,{},throughput,{},{},{}",
-                    opt.scenario,
-                    opt.name,
-                    opt.payload,
+                    scenario,
+                    name,
+                    payload,
                     (c as f64 / interval).floor() as usize
                 );
             }
         }
     });
 
-    let sub_info = SubInfo {
-        reliability: Reliability::Reliable,
-        mode: SubMode::Push,
-        period: None,
-    };
     let mut sub = session
-        .declare_subscriber(&reskey, &sub_info)
+        .subscribe(expr_id)
+        .reliable()
+        .push_mode()
         .await
         .unwrap();
 
-    while sub.receiver().recv().is_ok() {
+    while let Ok(_) = sub.receiver().recv() {
```

Comment:
If we don't use the destructed value, checking `is_ok` is enough.

Reply:
Will revert this line and resubmit again.